mono/mini/mini-runtime.c

```diff
@@ -2483,8 +2483,7 @@ mono_jit_free_method (MonoDomain *domain, MonoMethod *method)
 	if (mono_use_interpreter) {
 		mono_domain_jit_code_hash_lock (domain);
 		/* InterpMethod is allocated in the domain mempool */
-		if (mono_internal_hash_table_lookup (&info->interp_code_hash, method))
-			mono_internal_hash_table_remove (&info->interp_code_hash, method);
+		mono_internal_hash_table_remove (&info->interp_code_hash, method);
```

Comment:
Shouldn't we add a g_assert to the other call to mono_internal_hash_table_remove?

Reply:
fair enough. updated!